community-operators/opendatahub-operator/opendatahub-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,858 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: "AI/Machine Learning, Big Data"
+    description: Open Data Hub is a community effort
+    containerImage: quay.io/opendatahub/opendatahub-operator:v0.0.1
+    createdAt: 2019-04-02T:01:23:45Z
+    repository: https://gitlab.com/opendatahub/opendatahub-operator
+    support: Open Data Hub
+    certified: "false"
+    alm-examples: '[ { "kind": "OpenDataHub", "spec": { "jupyter-on-openshift": { "jupyterhub_config": "" }, "spark-operator": { "master_node_count": 1, "master_memory": "2Gi", "master_cpu": 2, "worker_node_count": 2, "worker_memory": "2Gi", "worker_cpu": 2 } }, "apiVersion": "opendatahub.io/v1alpha1", "metadata": { "name": "example-opendatahub" } } ]'
+  name: opendatahub-operator.v0.0.1
+  namespace: placeholder
+spec:
+  maturity: alpha
+  version: 0.0.1
+  apiservicedefinitions: {}
+  labels:
+    operated-by: opendatahub-operator
+  selector:
+    matchLabels:
+      operated-by: opendatahub-operator
+  customresourcedefinitions:
+    owned:
+    - kind: OpenDataHub
+      name: opendatahubs.opendatahub.io
+      version: v1alpha1
+      displayName: Open Data Hub
+      description: Deployment of components from the Open Data Hub community
+  description: "The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform, Ceph Object Storage, and Kafka/Strimzi integrating a collection of open source projects"
```

Comment:
Please note that we support multiline descriptions and markdown formatting:
```
  description: |
    The Open Data Hub is a ...
```

Reply:
This has been tested on OCP4 Preview. I'll update this PR with a better multiline description and I need to change the starting version